src/App.js

```diff
@@ -1,12 +1,218 @@
 import React from 'react';
 import './App.css';
+import Buttons from "./Components/Buttons";
 
 function App() {
+
+    const [countBanana, setCountBanana] = React.useState(0);
+    const [countAardbei, setCountAardbei] = React.useState(0);
+    const [countApple, setCountApple] = React.useState(0);
+    const [countKiwi, setCountKiwi] = React.useState(0);
+    const [firstname, setFirstname] = React.useState('');
```

Comment:
Het is handig om als default waarde `undefined` te nemen. Dat bied je de mogelijkheid om bijv. de submit button te disablen als er nog geen waarde is ingevuld. Je zou natuurlijk `=== ""` kunnen doen om te controleren, maar `=== undefined` vind ik zuiverder.